extensions/doctrine/Numeric/Rand.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace Numeric;
+use Doctrine\ORM\Query\AST\Functions\FunctionNode;
+use Doctrine\ORM\Query\Lexer;
+/**
+ * RandFunction ::= "RAND" "(" ")"
+ */
+class Rand extends FunctionNode
+{
+
+    public function parse(\Doctrine\ORM\Query\Parser $parser)
+    {
+        $parser->match(Lexer::T_IDENTIFIER);
+        $parser->match(Lexer::T_OPEN_PARENTHESIS);
+        $parser->match(Lexer::T_CLOSE_PARENTHESIS);
+    }
+
+    public function getSql(\Doctrine\ORM\Query\SqlWalker $sqlWalker)
+    {
+        return 'RAND()';
+    }
+}
+?>
```

Comment:
?> moet weg.

Daarbij is dit zelf gemaakt, of komt het van het internet af? Graag iets meer uitleg (dan wel een link naar waar het vandaan komt, dan wel korte uitleg wat de functies doen)